packages/core/src/JWEditor.ts

```diff
@@ -140,7 +156,7 @@ export class JWEditor {
      */
     execCommand(id: CommandIdentifier, args?: CommandArgs): void {
         this.dispatcher.dispatch(id, args);
-        this.renderer.render(this.vDocument, this.editable);
+        this.renderers[this.defaultRendererId].render(this.vDocument, this.editable);
```

Comment:
All renderers should be called ? So if I have installed, say, a PDFRenderer, it re-generates a new PDF at every change ? I would have expected it to not do anything unless I click on "Export to PDF" or whatever. It looks like we may need a separation between "live" renderers and "exports" renderer ?

Reply:
They should at least have a signal that tells them that the document has changed.
Maybe instead of calling a method `render`, we could call something like `changeDocument`.

Reply:
We have a confusion of term / name.
I think that it is necessary to make the difference between the signal to render them (informing that they can update), and the request to the editor to obtain the result of a specific rendering.
The `renderers` (plugin thereafter) know if they are displayed or not, and know if they update.
Clicking on 'save as' will be another command.